query03.sql

```diff
@@ -0,0 +1,31 @@
+CREATE INDEX phl_pwd_parcels__the__geom__32129__idx
+	ON phl_pwd_parcels
+	using GiST (ST_Transform(geom, 32129));
+  
+with joined as (
+	select s.stop_name,
+		p.address,
+		s.the_geom,
+		p.geom
+	from phl_pwd_parcels as p
+	left join septa_bus_stops as s
+		on ST_DWithin(
+            ST_Transform(p.geom, 32129),
+            ST_Transform(s.the_geom, 32129),
+            500
+        )
```

Comment:
I'm a big fan of this solution. Even though the solution with a `cross join lateral` is faster in PostGIS, this solution where you limit the stops under consideration is a good one for pretty much any DB engine (especially data warehouses that charge based on the amount of data processed).